include/device.h

```diff
@@ -21,6 +21,7 @@
 
 #include <init.h>
 #include <sys/device_mmio.h>
+#include <device_extern.h>
```

Comment:
Maybe comment that this is a generated file?

Reply:
Added comment.